lib/index.js

```diff
@@ -169,6 +169,21 @@ internals.addMethod = function (names, fn) {
     internals.addMethod(word, method);
 });
 
+internals.addMethod('error', function (/*type, message*/) {
```

Comment:
Considering the logic and the error strings are slightly different, I can't see any re-use.